### ISSUE TITLE
Training loss？

### ISSUE THREAD

**Issue by wzczc** [THUDM/ImageReward#106]
Thanks for your great work！But I have some questions about the training loss in the code. The paper mentions thr ranking loss:
![image](https://github.com/user-attachments/assets/3fd48374-45f9-42ea-81f7-c4f353c5726b)
but the code uses cross-entropy loss. 
![image](https://github.com/user-attachments/assets/0f5baa06-3db8-4ebc-bbe0-f0ceb966b183)
Are these two the same, or do they have similar optimization objectives?

**Comment by Chency-dear** [comment]
i have the same problem, why the targets be set to zero. why is it set [1,0] or [0,1] to distinguish the better and the worse

**Comment by TaylorJocelyn** [comment]
Hello, I have a question regarding the formula. I was wondering why the formula doesn't explicitly account for the difference between positive and negative samples. Could you kindly share your thoughts or insights on this? I would greatly appreciate it.
